README.rst

```diff
@@ -5,7 +5,7 @@ ONGR is a platform based on Symfony 2 framework which stands in front of your ap
 
 It is a separate system from your original application where you re-implement or move view and frontend related code to ONGR for high-performance content delivery. Product and other data is still managed by original backend and then continuously synchronized with new platform for cache and display.
 
-If you find some issues or great ideas how to improve the project, please create an issue in GitHub. Also everyone are more than welcome to contribute using pull requests. More information is in `contributing page </Resources/doc/contributing.rst>`_.
+If you find some issues or great ideas how to improve the project, please create an issue in GitHub. Also everyone are more than welcome to contribute using pull requests. More information is in `contributing page <https://github.com/ongr-io/ongr-handbook/blob/master/source/handbook/contributing/contributing.rst>`_.
```

Comment:
I am not entirely sure about these links. Do we intend the person reading this doc to leave our documentation page for github?